oak-core/src/test/java/org/apache/jackrabbit/oak/query/AbstractQueryTest.java

```diff
@@ -108,17 +108,21 @@ protected void createTestIndexNode() throws Exception {
         root.commit();
     }
 
-    protected static Tree createTestIndexNode(Tree index, String type)
-            throws Exception {
+    protected  static Tree createTestIndexNode(String indexName, Tree index, String type) {
```

Comment:
nit: remove extra space
```suggestion
    protected static Tree createTestIndexNode(String indexName, Tree index, String type) {
```

Reply:
done